build.ps1

```diff
@@ -10,14 +10,14 @@ Param (
 $NugetVersion       = "4.4.0"
 $UseExperimental    = $false
 $RootDir            = "${PSScriptRoot}"
-$ScriptFile         = "${RootDir}/build.cake"
-$BuildDir           = "${RootDir}/_build"
-$ToolsDir           = "${BuildDir}/tools"
-$PackagesDir        = "${BuildDir}/lib/nuget"
-$NugetExe           = "${ToolsDir}/NuGet/${NugetVersion}/nuget.exe"
-$PackagesConfigFile = "${RootDir}/packages.config"
+$ScriptFile         = "${RootDir}\build.cake"
+$BuildDir           = "${RootDir}\_build"
+$ToolsDir           = "${BuildDir}\tools"
+$PackagesDir        = "${BuildDir}\lib\nuget"
+$NugetExe           = "${ToolsDir}\NuGet\${NugetVersion}\nuget.exe"
+$PackagesConfigFile = "${RootDir}\packages.config"
 $CakeVersion        = (Select-Xml -Xml ([xml](Get-Content $PackagesConfigFile)) -XPath "//package[@id='Cake'][1]/@version").Node.Value
-$CakeExe            = "${PackagesDir}/Cake.${CakeVersion}/Cake.exe"
+$CakeExe            = "${PackagesDir}\Cake.${CakeVersion}\Cake.exe"
 
```

Comment:
Since Windows is fine with the forward slashes, but Linux treats them as escape characters, it makes sense to me to use forward slashes to make the two build scripts easy to keep in sync. Is there a particular reason to switch to backslashes other than "that's how Windows users see paths?"

Reply:
Yeah, this change isn't really necessary as both forward- and backslashes are valid path separators on Windows.